devops/docker/rhel-7-amd64/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# ubi7 images are missing RPM devtools, copy from centos7
+FROM centos:centos7 as centos
+RUN yum update -y && yumdownloader rpm-build && yumdownloader elfutils
+
+# TODO: Add MSFT mirror
```

Comment:
See the other comment. Applies everywhere. No "MSFT" in public GitHub repo, please.

Reply:
addressed